### PR TITLE
Prevents changes from being overwritten in updates

### PR DESCRIPTION
**Pull request by PhoenixPeca**
Ref: [#370](https://github.com/mailcow/mailcow-dockerized/pull/370)

### CODE REVIEW

**Comment by mkuron** [comment]
It shouldn't be necessary to copy and paste the entire config. This should suffice:
```
$config['useEASforOutlook'] = 'no';
```

**Comment by mkuron** [comment]
That doesn't work. You are overwriting `$config` completely, so things like the server name wouldn't be available anymore. You need to actually set individual members as I suggested before.

File: docs/u_e-mailcow_ui-config.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Several configuration parameters of the mailcow UI can be changed by creating a file `data/web/inc/vars.local.inc.php` which overrides defaults settings found in `data/web/inc/vars.inc.php`.
+Several configuration parameters of the mailcow UI can be changed by creating a file `data/web/inc/vars.local.inc.php` which overrides defaults settings found in `data/web/inc/vars.inc.php`, among others include `data/web/autodiscover.php`.
```
Comment (mkuron) [inline docs/u_e-mailcow_ui-config.md]: Should be "including" instead of "include".

Comment (PhoenixPeca) [inline docs/u_e-mailcow_ui-config.md]: cool thanks

**Comment by PhoenixPeca** [comment]
This issue has been managed #30

**Pull request closed without merging**